honeybee_radiance_command/rpict.py

```diff
@@ -77,10 +77,16 @@ def to_radiance(self, stdin_input=False):
 
         command_parts = [self.command, self.options.to_radiance(), self.octree]
         cmd = ' '.join(command_parts)
+
+        if stdin_input and self.view and self.output:
+            cmd = '%s | %s' % (self.view, cmd)
```

Comment:
This doesn't look right. If the view information is provided from stdin then the view from the command should not be used.